src/syscalls/native.rs

```diff
@@ -590,14 +590,53 @@ pub struct SpawnArgs {
 /// The parent process calls the Spawn system call, which creates a new process (a child process) that is an
 /// independent ckb-vm instance. It's important to note that the parent process will not be blocked by the child
 /// process as a result of this syscall.
-/// Note: available after ckb2023.
+/// Note: available after ckb 2nd hardfork.
 ///
 /// # Arguments
 ///
 /// * `index`, `source`, `bounds` and `place` - same as exec.
 /// * `spgs` - spawn arguments.
 ///
 /// Returns success or a syscall error.
+///
+/// # Scheduler Algorithm
```

Comment:
Better name this to scheduler algorithm v1, as latter hardforks might introduce v2, v3, et.